base/BUILD.gn

```diff
@@ -1836,6 +1748,74 @@ component("base") {
     ]
   }
 
+  # TODO: b/384652502 - Cobalt: Fix compiler errors building hermetically.
+  if (is_cobalt && is_cobalt_hermetic_build) {
+    sources -= [
```

Comment:
I was suggesting multiple `sources -= [...]` blocks in the conditional to have clear partitioning still, but this is alright too